external/scintilla/src/EditView.cxx

```diff
@@ -880,7 +880,11 @@ void EditView::DrawEOL(Surface *surface, const EditModel &model, const ViewStyle
 		if (background.isSet) {
 			surface->FillRectangle(rcSegment, background);
 		} else if (line < model.pdoc->LinesTotal() - 1) {
+			// Note(Bonzomatic): ULGY HACK TO REMOVE CHAR BACKGROUND ON END OF LINES
```

Comment:
"ULGY"? :)

Reply:
Hahaha omg, it's even more accurate like that :D